src/main/java/core/basesyntax/service/FigurePainter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FigurePainter {
```

Comment:
```suggestion
public interface Painter {
```
Интерфейс могут реализовать не только фигуры.